guide/contribute/illustration-guidelines.md

```diff
@@ -0,0 +1,161 @@
+
+Each markdown file should contain a comment at the top that include links to these source files. For [example](https://raw.githubusercontent.com/BitcoinDesign/Guide/master/guide/getting-started/common-user-flows.md):
+
+{% raw %}
+```
+<!--
+
+Editor's notes
+
+This page provides an overview to...
+
+Illustration sources
+
+- https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?node-id=862%3A2982
+
+-->
+```
+{% endraw %}
+
+## File folders
+
+All images in the guide are located in the `assets/images/guide` folder. Create a subfolder that matches the URL of your page and place your images there.
+
+For example, the URL for this page is:
+
+`/guide/contribute/illustration-guidelines/`
+
+Images for this page are located in:
+
+`assets/images/guide/contribute/illustration-guidelines/`
+
+## File names
+
+Keep your file names in lowercase and avoid any special characters (including spaces). Keep them short and descriptive.
+
+**Don't**
+
+`My übercool image.PNG`
+
+**Do**
+
+`transaction-confirmation.png`
+
+## JPG, PNG or GIF?
+
+For static images, use JPG or PNG. The two formats use different compression methods. PNG files are better for images with flat colors, while JPGs can compress photos and heavily textures visuals better. When in doubt, export both file types and see which one looks best at an acceptable file size.
+
+For short animations, use GIFs. Be aware that GIFs can easily bloat in file size and have limited color depth.
+
+## Export images at the right sizes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/contribute/illustration-guidelines/image-guide.png"
+   retina = "/assets/images/guide/contribute/illustration-guidelines/image-guide@2x.png"
+   alt-text = "Example image"
+   width = 400
+   height = 608
+   layout = "float-right-desktop"
+%}
+
+This ensures images render crisp and clear while keeping file download size and speed reasonable.
+
+Overall, there are five layout options for images, each one resulting in a different image size. Aditionally, desktop and mobile layout logic differs, and we also need to support regular and retina screen resolutions. Result is that each image needs to be exported at four different sizes. For examples, view the [Formatting](/guide/contribute/formatting/#images) page.
+
+Pay attention to the image width when you export. Image height is up you, based on the image content.
+
+Image width specifications are as follows, with widths provided in pixels. 2x images are for retina screens and always double the size of the regular image.
+
+</div>
+
+| Type | Purpose | width
+|:-|:-|:-|
+| 1. Header banner | Desktop | 1600 |
```

Comment:
@GBKS this size is not only used on the header. For instance here https://bitcoin.design/guide/private-key-management/schemes/#picking-a-scheme-for-your-product

Reply:
Yep, that page is doing it wrong.